its/src/test/java/com/sonar/it/scanner/msbuild/utils/TestUtils.java

```diff
@@ -70,7 +70,7 @@ public class TestUtils {
   private static String token = null;
 
   public static final Long TIMEOUT_LIMIT = 60 * 1000L;
-  public static final String MSBUILD_DEFAULT_PATH = "C:\\Program Files (x86)\\Microsoft Visual Studio\\2017\\Enterprise\\MSBuild\\15.0\\Bin\\MSBuild.exe";
+  public static final String MSBUILD_DEFAULT_PATH = "C:\\Program Files\\Microsoft Visual Studio\\2022\\Professional\\MSBuild\\Current\\Bin\\MSBuild.exe";
```

Comment:
```suggestion
  public static final String MSBUILD_DEFAULT_PATH = @"C:\Program Files\Microsoft Visual Studio\2022\Professional\MSBuild\Current\Bin\MSBuild.exe";
```

Reply:
Java doesn't have verbatim strings.

:crying-in-java: